handler/oauth2/flow_authorize_implicit.go

```diff
@@ -94,12 +94,20 @@ func (c *AuthorizeImplicitGrantTypeHandler) IssueImplicitAccessToken(ctx context
 	if err := c.AccessTokenStorage.CreateAccessTokenSession(ctx, signature, ar.Sanitize([]string{})); err != nil {
 		return errors.WithStack(fosite.ErrServerError.WithCause(err).WithDebug(err.Error()))
 	}
+	if ar.GetRequestForm().Get("response_mode") == "form_post" {
```

Comment:
It looks like the two if statements are copies of one another. Can we avoid duplication here?

Reply:
yes it can be improved